resources/application_resource.go

```diff
@@ -42,10 +44,18 @@ func (a Application) MarshalJSON() ([]byte, error) {
 		Metadata: a.Metadata,
 	}
 
+	relationShips := Relationships{}
+
 	if a.SpaceGUID != "" {
-		ccApp.Relationships = Relationships{
-			constant.RelationshipTypeSpace: Relationship{GUID: a.SpaceGUID},
-		}
+		relationShips[constant.RelationshipTypeSpace] = Relationship{GUID: a.SpaceGUID}
+	}
+
+	if a.CurrentDropletGUID != "" {
+		relationShips[constant.RelationshipTypeCurrentDroplet] = Relationship{GUID: a.CurrentDropletGUID}
+	}
+
+	if len(relationShips) > 0 {
```

Comment:
Do we really need this if statement here? if we could I would prefer to have it removed and always set relationships in the struct